### ISSUE TITLE
Swamp of Sorrow Mobs Not Spawned

### ISSUE THREAD

**Issue by GM-Martok** [gamefreedomgit/Maelstrom#6084]
[//]: # (REMBEMBER! Add links to things related to the bug using for example:)
[//]: # (https://www.wowhead.com/cata/item=11205/formula-enchant-gloves-advanced-herbalism#dropped-by)

**Description:**
Player was attempting to obtain Formula Enchant Gloves: Advanced Herbalism and noticed there were no mobs on the map. After spending some time looking, a ticket was opened. GM Aeradel spoke with GM Isa to see if there were any known issues. She suggested to submit a Bug Report. GM Chronus and GM Martok also confirmed.

 The following mobs were confirmed to not be present:

Swamwalker
Tangled Horror
Swampwalker Elder
Mire Lord
Molt Thorn

**Comment by PanJanuszek** [comment]
**"The pattern is still in game but extremely difficult to get. It's in the Swamp of Sorrows but that's all I can really say.
Given the difficulty I have submitted a report to our quality assurance team to investigate as this may be unintended given the mobs were removed and a slight oversight, though I cannot really say.They also may have wanted to make this enchant extremely rare. ;)
Any updates will be communicated through the forums at http://us.battle.net/wow/en/forum/ .
As well, any more information on the enchant can be found at [www.wowhead.com/cata](https://www.wowhead.com/cata) =)
Thank you for your time.
Cheers!"**

This is the response from blizzard game master. That's right there is only one living NPC in world who can drop this recipe, ... and it is rare spawn.

https://www.wowhead.com/cata/npc=14448/molt-thorn

 I can agree to add 100% drop recipe for this rare spawn, because it's only one on map.
